### ISSUE TITLE
Staging Review - Accessibility Feedback - Converted component: Breadcrumbs

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#2134]
## Description
Breadcrumbs
* Should: This is a departure from the USWDS, but the last item in the breadcrumbs indicating the current page should be a link with aria-current="page" applied to the a element. That makes it consistent with the other elements in the nav, including being focusable as screen reader users tab through the elements. @rsmithadhoc's comment on this has a good thought process to back up this decision, and it's supported by the W3C's ARIA Authoring Practices Guide and some examples from Scott O'Hara and Léonie Watson. In terms of styling, I would suggest we still maintain material honesty (unlike the W3C and Scott O'Hara examples): the current page element should still look like a link. It might make sense to style the link differently (bold? different color? different underline style?) but it should still be recognizably linky. Hopefully that's an improvement that can also be contributed back to the USWDS. Marking this as a should since the component as it is now is semantically correct and consistent with other sources. But I think it's a strong should!

## Details
This is the feedback specific to Breadcrumbs from the staging review. https://github.com/department-of-veterans-affairs/va.gov-team/issues/64284

## Tasks
- [ ] Address the 'should' feedback
- [ ] Have accessibility specialist review and sign off 
- [ ] Make updates as needed 

## Acceptance Criteria
- [ ] Component is updated

**Comment by caw310** [comment]
Closing as this was addressed by 
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2117
